types/block.go

```diff
@@ -43,7 +43,7 @@ const (
 	MaxOverheadForBlock int64 = 11
 )
 
-// Block defines the atomic unit of a Tendermint blockchain.
+// Block defines the atomic unit of a Ostracon blockchain.
```

Comment:
*an* Ostracon
Please check out all the articles in Ostracon by searching for "a Ostracon".